jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/JRubyExec.groovy

```diff
@@ -19,19 +19,25 @@ import org.gradle.util.CollectionUtils
 class JRubyExec extends JavaExec implements JRubyExecTraits {
 
     static final String JRUBYEXEC_CONFIG = 'jrubyExec'
+    static final String JAR_DEPENDENCIES_VERSION = '0.1.15'
 
     static void updateJRubyDependencies(Project proj) {
         proj.dependencies {
             jrubyExec "org.jruby:jruby-complete:${proj.jruby.execVersion}"
-            jrubyExec "org.bouncycastle:bcprov-jdk15on:${proj.jruby.bouncycastleVersion}"
+            if (proj.jruby.execVersion.startsWith("1.7.1")) {
```

Comment:
Will you link to a changelog entry or something explaining why we do not need to inject a dependency on versions of JRuby >= `1.7.20`